css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,51 @@
+$breadcrumb-font-size: $document-font-size !default;
+$breadcrumb-spacing-horizontal-start: 0.5em !default;
+$breadcrumb-spacing-horizontal-end: 0.25em !default;
+$breadcrumb-padding-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+$breadcrumb-color: $hyperlink !default;
+
+.breadcrumbs {
+	font-size: $breadcrumb-font-size;
+
+	.breadcrumb-overflow {
```

Comment:
Thinking this would not be needed if breadcrumbs was limited to list of crumbs and did not contain the overflow. You'd of course need a modifier to determine whether there should be a leading slash which would need to be added programmatically when overflow occurred at the same time as the overflow menu... thoughts?

Reply:
something along the lines of this?

`.breadcrumbs`
```
bla / bla / bla /
```
`.breadcrumbs.breadcrumbs-overflowed`
```
/ bla / bla / bla /
```

Reply:
Thoughts are that the current version will allow us to be a little more exacting in terms of spacing on design consistency - and that `.breadcrumbs.breadcrumbs-overflowed` is an less easily understandable concept than an optional subcomponent.

Reply:
I'd also be open to making the `breadcrumbs-overflow` some day, so as to even include the overflow menu and details - but seems out of scope for this PR.